### PR TITLE
Add ToC header

### PR DESCRIPTION
**Pull request by awong503**
edit for a visually appealing ToC header for links to the other sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ This repository powers multiple language support for
 [CodeCrafters](https://codecrafters.io) challenges.
 
 <!-- toc -->
+## Table of Contents
```
Comment (rohitpaulk) [inline README.md:5]: @awong503 thanks for this! Minor change: Could you place this on top of the `<!-- toc -->` marker please? The table of contents is generated automatically using [this command](https://github.com/codecrafters-io/languages/blob/d883305d2532325a57696440045efd496de6a488/Makefile#L13-L14), and it'll replace anything inside the `<!-- toc -->` markers.

Comment (awong503) [reply README.md:4]: Yup, will do! Thanks for pointing that out.

**Comment by awong503** [comment]
Complete requested change.

- moved header above the `<!-- toc -->` marker

**Comment by rohitpaulk** [comment]
Thanks @awong503!

**Pull request merged**